### DISCUSSION TITLE
Important: Discontinuation of Meowix - Immediate Transition to a New Distro Advised

### DISCUSSION THREAD

**Discussion by Nekosis** [Meowix-Linux/Meowix-ISO#11] [Announcements]
Hey everyone,

This is Nekosis. I've made a tough call—I'm halting the development of Meowix effective immediately. I know this might be unexpected. We haven't had a release for the past couple of months, which doesn't align with our usual monthly schedule. Between school and everything else happening in my life, I've found it increasingly hard to allocate time for its development. Moreover, I don't feel Meowix meets the high standards needed for serious usage. I take responsibility for promoting it in a way that didn't accurately represent its experience due to significant instability issues, and for that, I'm sorry.

For anyone still using Meowix, I apologize for any inconvenience caused. I strongly recommend exploring similar distros like [Archcraft](https://archcraft.io/) or [EndeavourOS](https://endeavouros.com/), both offering preconfigured tiling window manager options. **Please transition away from Meowix as soon as possible.** I've made numerous design mistakes, and I expect any Meowix setup to be partially dysfunctional within a year.

I've been gradually shifting away from Linux as it becomes less central in my life. I find myself using the Windows side of my dual-boot more due to complications I've faced with Arch. I simply want things to function smoothly, and Windows offers that. Even if it means sacrificing FOSS principles... so be it. I just want to live my life without the FSF or OSI dictating my choices. I might even take a break from GitHub and the FOSS community due to this stress.

I recognize the importance of advocating for a better online and computing environment—where data is genuinely secure, ads are minimal, the right to repair is respected, and users have full control over their PCs. But at this moment, these ideals aren't my top priority. Other aspects of my life demand more attention as a high school teenager. Starting something during summer and then struggling to keep up with it once school starts? Who could have foreseen that?

Nonetheless, thank you immensely for supporting this quirky, temperamental project hastily put together by a 14-year-old in just a few months.

You folks really put your trust in anything, huh?

*- Nekosis*

**Comment by MeBadDev** [comment]
Thank you for the awesome distro! Im suprised that you're only 14 years old XD
